CI fails at installing phantomjs@1.9.16

Currently where build_and_test falls flat:

```
> phantomjs@1.9.16 install /home/runner/work/sandstorm/sandstorm/tests/node_modules/phantomjs
> node install.js

fs.js:35
} = primordials;
    ^

ReferenceError: primordials is not defined
    at fs.js:35:5
    at req_ (/home/runner/work/sandstorm/sandstorm/tests/node_modules/natives/index.js:143:24)
    at Object.req [as require] (/home/runner/work/sandstorm/sandstorm/tests/node_modules/natives/index.js:55:10)
    at Object.<anonymous> (/home/runner/work/sandstorm/sandstorm/tests/node_modules/phantomjs/node_modules/graceful-fs/fs.js:1:37)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14)
    at Module.require (internal/modules/cjs/loader.js:1044:19)
    at require (internal/modules/cjs/helpers.js:77:18)
npm WARN sandstorm-tests@0.0.1 No license field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! phantomjs@1.9.16 install: `node install.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the phantomjs@1.9.16 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2020-03-16T00_34_20_017Z-debug.log
Makefile:165: recipe for target 'test' failed
make: *** [test] Error 1
```

--- a/tests/package.json
+++ b/tests/package.json
@@ -21,7 +21,6 @@
     "mailparser": "0.5.1",
     "selenium-standalone": "^6.15.4",
     "simplesmtp": "0.3.35",
-    "spacejam": "1.2.2",
     "underscore": "^1.9.1"
   }
 }

